### ISSUE TITLE
[JOSS Review] Documentation

### ISSUE THREAD

**Issue by gvegayon** [JonasMoss/kdensity#48]
Part of JOSS review openjournals/joss-reviews#1566

- [x] The S3 method lines.kdensity has no documentation. Since the function has a special set of arguments (like in the vignette), I would suggest to include a manual for the method as well.

- [x] The members of the `kdensity` class object (enviornment) are not fully documented. I understand that the user may not need to be fully aware of its contents, but I just want to make sure that this is something explicitly done by the developer (which would be OK).

- [x] Although the developer mentions it, I think it would be good for the package to enumerate the available bandwidth selectors explicitly as items in the manual. Since you are using `roxygen2`, you could use the `aliases` tag to add them to the documentation by means of the name of the method (not the actual name of the function which are actually abbreviations).

**Comment by JonasMoss** [comment]
I've responded to the two first points in 46d9d6670380e3d09030b179801857e123213ac0. I can't document all the members of the environment since some are internal, but I restricted the elements [[]] and $ can access. Thanks for spotting this!

I'm not sure about enumerating all the bandwidth selectors since it would overpopulate the documentation. (I prefer it lean!) And it would pressure me to enumerate the documentation for all the starts and kernels too, further overpopulating the documentation.
